File: global/blade/app/utils/Task.php
<?php

namespace App\Utils;

class Task{
    private string $nome;

    private bool $concluida;

    public function __construct(string $nome){
        $this->nome = $nome;
        $this->concluida = false;
    }
    
    public function getNome(): string {
        return $this->nome;
    }

    public function setNome(string $nome): void {
        $this->nome = $nome;
    }

    public function getConcluida(): bool {
        return $this->concluida;
    }

    public function setConcluida(bool $concluida): void {
        $this->concluida = $concluida;
    }
}

File: backend/djangopy/todolist/utils/task.py
class Task:
    def __init__(self, nome: str):
        self.nome = nome
        self.concluida = False

    def get_nome(self) -> str:
        return self.nome

    def set_nome(self, nome: str) -> None:
        self.nome = nome

    def get_concluida(self) -> bool:
        return self.concluida

    def set_concluida(self, concluida: bool) -> None:
        self.concluida = concluida

    def to_dict(self):
        return {
            'nome': self.nome,
            'concluida': self.concluida,
        }
